game/scripts/vscripts/abilities/oaa_divided_we_stand.lua

```diff
@@ -0,0 +1,317 @@
+            if ability then
+                for _, meepo in pairs(GetAllMeepos(mainMeepo)) do
+                    local cloneAbility = meepo:FindAbilityByName(ability.GetAbilityName(ability))
+                    if ability:GetLevel() > cloneAbility:GetLevel() then
+                        cloneAbility:SetLevel(ability.GetLevel(ability))
+                        meepo:SetAbilityPoints(meepo.GetAbilityPoints(meepo)-1)
+                    end
+                    if ability:GetLevel() < cloneAbility:GetLevel() then
+                        ability:SetLevel(cloneAbility:GetLevel())
+                        mainMeepo:SetAbilityPoints(mainMeepo:GetAbilityPoints()-1)
+                    end
+                end
+            end
+        end
+    end
+end
+function GetAllMeepos(caster)
+    if caster.meepoList then
+        return caster.meepoList
+    else
+        return {caster}
+    end
+end
+function MeepoExperience(filterTable)
+    local PID = filterTable.player_id_const
+    local reason = filterTable.reason_const
+    local experience = filterTable.experience
+    local hero = PlayerResource.GetSelectedHeroEntity(PlayerResource,PID)
+    if (hero and hero.HasAbility(hero,"meepo_divided_we_stand")) and (reason~=DOTA_ModifyXP_Unspecified) then
+        filterTable.experience=0
+        hero.AddExperience(hero,experience,DOTA_ModifyXP_Unspecified,true,false)
+    end
+    return filterTable
+end
+function MeepoOrderFilter(filterTable)
+    local entindex_ability = filterTable.entindex_ability
+    local sequence_number_const = filterTable.sequence_number_const
+    local queue = filterTable.sequence_number_const
+    local units = filterTable.units
+    local entindex_target = filterTable.entindex_target
+    local position = Vector(filterTable.position_y,filterTable.position_y,filterTable.position_z)
+    local order_type = filterTable.order_type
+    local issuer_player_id_const = filterTable.issuer_player_id_const
+    local ability = EntIndexToHScript(entindex_ability)
+    local target = EntIndexToHScript(entindex_target)
+    for _, entindex_unit in pairs(units) do
+        local unit = EntIndexToHScript(entindex_unit)
+        if unit.HasModifier(unit,"modifier_meepo_divided_we_stand_oaa") and (unit~=PlayerResource.GetSelectedHeroEntity(PlayerResource,unit.GetPlayerOwnerID(unit))) then
+            if order_type==DOTA_UNIT_ORDER_PICKUP_ITEM then
+                return false
+            end
+        end
+        if ((target and target.HasModifier) and target.HasModifier(target,"modifier_meepo_divided_we_stand_oaa")) and (target~=PlayerResource.GetSelectedHeroEntity(PlayerResource,target.GetPlayerOwnerID(target))) then
+            if order_type==DOTA_UNIT_ORDER_GIVE_ITEM then
+                return false
+            end
+        end
+    end
+    return true
+end
+if IsServer() then
+    GameRules.GetGameModeEntity(GameRules).SetExecuteOrderFilter(GameRules.GetGameModeEntity(GameRules),function(f,filterTable)
```

Comment:
Will this overwrite our existing filter? I haven't tested it... We have a filter manager for registering new filters of the same type, pretty sure that's why it exists